fe/fe-core/src/main/java/com/starrocks/sql/ast/CTERelation.java

```diff
@@ -7,7 +7,8 @@
 import java.util.List;
 
 public class CTERelation extends Relation {
-    private final int cteId;
+    // @Note: don't join hash method, maybe multi-change
+    private int cteId;
```

Comment:
I think the cteId should be the unique identifier for the CTERelation, if the  cte id change, we should new a CTERelation.

Reply:
I see, but it's too difficult.